setup.py

```diff
@@ -11,13 +12,46 @@
 from rpm_py_installer.version import VERSION
 
 
+def run_cmd(cmd):
+    """Run a command."""
+    exit_status = 0
+    message = ''
+    message_format = 'Command: {0}'
+
+    # The subprocess.run() function was added in Python 3.5.
+    # https://docs.python.org/3.5/library/subprocess.html
+    if sys.version_info > (3, 5):
+        p = subprocess.run(
+            cmd.split(' '), stdout=subprocess.PIPE, stderr=subprocess.PIPE
+        )
+        exit_status = p.returncode
+
+        message_format = "\n".join(
+            [
+                message_format,
+                'Return Code: [{1}]',
+                'Stdout: [{2}]',
+                'Stderr: [{3}]'
+            ]
+        )
```

Comment:
I noticed we can simplify this part to set the multi lines string message without `join`command.

Reply:
I think the `os.linesep` is better than the `\n`.

> https://docs.python.org/3/library/os.html

```
os.linesep

    The string used to separate (or, rather, terminate) lines on the current platform. This may be a single character, such as '\n' for POSIX, or multiple characters, for example, '\r\n' for Windows. Do not use os.linesep as a line terminator when writing files opened in text mode (the default); use a single '\n' instead, on all platforms.
````